src/components/FoodBox.js

```diff
@@ -0,0 +1,23 @@
+import { Card, Col, Button } from "antd";
+
+
+function FoodBox({foodItem, deleteFoodItem}) {
+    return ( 
+    <Col>
```

Comment:
Just choose a span, and that's it.

```suggestion
    <Col span={8}>
```